insights/2014-04-01/swagger/insightsClient_UsageMetrics.json

```diff
@@ -140,6 +136,14 @@
       "description": "The identifier of the resource.",
       "x-ms-parameter-location": "method",
       "x-ms-skip-url-encoding": true
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client Api Version.",
+      "x-ms-parameter-location": "method"
```

Comment:
Usages is an exceptional case. Because of the way the call is done we need the api-version to be an explicit parameter of the call. It cannot be implicit and taken from the api-version in the header of the spec file. 
Is there any way to generate code that leaves this api-version as an explicit parameter of the call?